lib/converters/v2.1.0/converter-v21-to-v1.js

```diff
@@ -95,9 +95,8 @@ module.exports = {
             authOptions = { excludeNoauth: true },
             units = ['order', 'folders_order', 'folders', 'requests'],
             varOpts = { fallback: options && options.env },
-            id = _.get(collection, 'info._postman_id') || _.get(collection, 'info.id'),
             newCollection = {
-                id: options && options.retainIds && id ? id : util.uid(),
```

Comment:
`id` fields should honor `retainIds`. Only the `_postman_id` should be retained.

Reply:
Addressed